lib/ansible/modules/cloud/amazon/rds_snapshot_facts.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: rds_snapshot_facts
+version_added: "2.4"
+short_description: obtain facts about one or more RDS snapshots
+description:
+  - obtain facts about one or more RDS snapshots
```

Comment:
@s-hertel in the [new version of this PR](https://github.com/ansible/ansible/pull/26599) I added documentation of the lack of support for Aurora.  I think it would be obviously better to fix this.